dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/TaskExecutionContext.java

```diff
@@ -172,12 +172,6 @@ public class TaskExecutionContext implements Serializable {
      */
     private String environmentConfig;
 
-    /**
-     * definedParams
-     * // todo: we need to rename definedParams, prepareParamsMap, paramsMap, this is confusing
-     */
-    private Map<String, String> definedParams;
```

Comment:
hi @bajiaolong ``` definedParams  ``` Is this parameter only used for Flink type tasks?

Reply:
> hi @bajiaolong `definedParams ` Is this parameter only used for Flink type tasks?

In addition to `flink`, there are also `MapReduce` and `Pigeon` tasks in use. But this parameter seems to have no effect, because I didn't see the task place setting it. The `Pigeon` task has test cases, so I revised them together.